.travis.yml

```diff
@@ -4,6 +4,7 @@ sudo: required
 language: go
 go:
   - 1.9.x
+  - 1.10.x
```

Comment:
From travis docs: "Note that, in order to choose Go 1.10, you must use go: "1.10" (a string), not go: 1.10 (a float). Using a float results in the use of Go 1.1."

source: https://docs.travis-ci.com/user/languages/go/

Reply:
`1.10` is different from `1.10.x`. https://travis-ci.org/contribsys/faktory/jobs/346889794